africanlii/context_processors.py

```diff
@@ -0,0 +1,90 @@
+LIIS = [
+    {
+        "name": "EswatiniLII",
+        "country": "Eswatini",
+        "url": "https://eswatinilii.org",
+        "logo": "images/liis/eswatinilii.png",
+    },
+    {
+        "name": "GhaLII",
+        "country": "Ghana",
+        "url": "https://ghalii.org",
+        "logo": "images/liis/ghalii.png",
+    },
+    {
+        "name": "Kenya Law",
+        "country": "Kenya",
+        "url": "http://kenyalaw.org",
+        "logo": "images/liis/kenyalaw.png",
+    },
+    {
+        "name": "LesothoLII",
+        "country": "Lesotho",
+        "url": "https://lesotholii.org",
+        "logo": "images/liis/lesotholii.png",
+    },
+    {
+        "name": "MalawiLII",
+        "country": "Malawi",
+        "url": "https://malawilii.org",
+        "logo": "images/liis/malawilii.png",
+    },
+    {
+        "name": "NamibLII",
+        "country": "Namibia",
+        "url": "https://namiblii.org",
+        "logo": "images/liis/namiblii.png",
+    },
+    {
+        "name": "SierraLII",
+        "country": "Sierra Leone",
+        "url": "https://sierralii.org",
+        "logo": "images/liis/sierralii.png",
+    },
+    {
+        "name": "SeyLII",
+        "country": "Seychelles",
+        "url": "https://seylii.org",
+        "logo": "images/liis/seylii.png",
+    },
+    {
+        "name": "LawLibrary",
+        "country": "South Africa",
+        "url": "https://lawlibrary.org.za",
+        "logo": "images/liis/lawlibrary.png",
+    },
+    {
+        "name": "TanzLII",
+        "country": "Tanzania",
+        "url": "https://tanzlii.org",
+        "logo": "images/liis/tanzlii.png",
+    },
+    {
+        "name": "ULII",
+        "country": "Uganda",
+        "url": "https://ulii.org",
+        "logo": "images/liis/ulii.png",
+    },
+    {
+        "name": "ZambiaLII",
+        "country": "Zambia",
+        "url": "https://zambialii.org",
+        "logo": "images/liis/zambialii.png",
+    },
+    {
+        "name": "ZanzibarLII",
+        "country": "Zanzibar",
+        "url": "https://zanzibarlii.org",
+        "logo": "images/liis/zanzibarlii.png",
+    },
+    {
+        "name": "ZimLII",
+        "country": "Zimbabwe",
+        "url": "https://zimlii.org",
+        "logo": "images/liis/zimlii.png",
+    },
+]
+
+
+def show_liis(request):
```

Comment:
this isn't doing anything and is just weird. Just use `LIIS` in the places where this is called.